bundles/mapping/infobox/plugin/openlayerspopup/OpenlayersPopupPlugin.ol.js

```diff
@@ -361,9 +373,53 @@ Oskari.clazz.define(
                     'class': key.iconCls,
                     'style': key.styles
                 });
-                headerWrapper.append(additionalButton);
+                popupHeaderTitle.append(additionalButton);
             });
 
+            headerWrapper.append(popupHeaderTitle);
+
+            // render coordinates to gfi header
+            if (showCoordinates) {
+                let mapModule = this.getMapModule();
+                let loc = Oskari.getLocalization('coordinatetool');
```

Comment:
This assumes that we have coordinatetool bundle as part of the app. Maybe just copy/paste loc.display.compass.lat/lon/n/e to the shared oskariui locale-files instead (https://github.com/oskariorg/oskari-frontend/tree/master/bundles/framework/oskariui/resources/locale). Or move and modify coordinate tool to use these as well?

Reply:
Oh, the crs part is needed as well. But oskariui could have something like this:
```
            "coordinates": {
                "lon": "Lon",
                "lat": "Lat",
                "n": "N",
                "e": "E",
                "p": "N",
                "i": "E", 
                "crs": {
                  "EPSG:3067": "ETRS89-TM35FIN coordinates",
                  "EPSG:3575": "North Pole LAEA Europe coordinates",
                  "EPSG:3857": "WGS 84 / Pseudo-Mercator coordinates",
                  "default": "{crs} coordinates"
              },
            },
```